user_gjrg_optins.html

```diff
@@ -0,0 +1,43 @@
+<div id="id_action_gjrg_partner_box" class="input-radio ak-input-type-action text-white" style="display: none;">
+  <label id="gjrg_partner_question" for="id_action_gjrg_partner_optin"><br>
+  {% filter ak_text:"gjrg_partner_question" %}
+    Would you like to receive email updates from <span class="gjrg_partner_name"></span>, a Global Just Recovery Gathering partner? <a id="gjrg_partner_privacy_link" target="_blank" href="">(Privacy Policy)</a>
+  {% endfilter %}
+  </label>
+  <span class="ak-userfield-radio-set">
+    <label class="ak-userfield-radio-choice" data-children-count="1">
+      <input id="id_action_gjrg_partner_optin" name="action_gjrg_partner_optin" type="radio" value="Y">
+      {% filter ak_text:"gjrg_partner_radio_label" %}
+        Yes, <span class="gjrg_partner_name"></span> can keep me updated
+      {% endfilter %}
+    </label>
+  </span>
+
+</div>
+
+<script>
+// Pull in partners object from additional html
+// Pull in query key from URL
+  // console.log('partners', window.gjrgPartners);
+  const urlParams = new URLSearchParams(window.location.search);
+  const partners = window.gjrgPartners ? window.gjrgPartners : null;
+  const partnerParam = urlParams.get('source') ? urlParams.get('source').split('.')[1] : "";
+  const partnerOptIn = document.querySelector('#id_action_gjrg_partner_box');
+  const partnerInputLabel = document.querySelector('#id_action_gjrg_partner_box label');
+  const partnerPrivacy = document.querySelector('#gjrg_partner_privacy_link');
```

Comment:
This is a nit for sure but why not use jQuery syntax instead of Vanilla JS because we have access to it? Open to hear your thoughts

Reply:
Hey @kimani-ndegwa you're quite right; I can refactor it to use jQuery. I guess I just missed using Vanilla JS 😅

Reply:
I'll make the changes and let you know when it's ready